circuitbreaker/mock_circuitbreaker.py

```diff
@@ -0,0 +1,44 @@
+import mockredis
+
+UNHEALTHY_ENDPOINTS = []
+
+
+class MockCircuitBreaker(object):
+    def __init__(self, endpoints, redis_config=None, logger_config=None):
+        self.redis_config = redis_config
+        self.redis = self.set_up_redis()
+        self.logger_config = logger_config
+        self._endpoints = endpoints
+        self._endpoint = None
+        self._complete = True
+        self.unhealthy_endpoints = UNHEALTHY_ENDPOINTS
```

Comment:
And since we're keeping track of unhealthy endpoints in the `MockCircuitBreaker` class here, do we really need the global at all? Just set this as an empty list to begin with, and append to it immediately after the `MockCircuitBreaker` object is instantiated, before calling `get_next_endpoint`. Or even create a method like `add_unhealthy_endpoint` that takes a string argument and appends it to the list so we don't have to directly interact with an internal list.